### PR TITLE
[FE] Add verification period

### PR DESCRIPTION
**Pull request by jkbktl**
This PR adds verification period to FE part of application both to desktop and mobile version.

- [x] add initial amount as max by default
- [x] add unverified votes to chart

Ready to test.

### CODE REVIEW

**Comment by jkbktl** [comment]
@prichodko @osmaczko guys, please check it out and test it if it works as you expect when you have time, I'll test it more thoroughly myself tomorrow

**Comment by prichodko** [comment]
closes #7

File: README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ Proposed api is available [here](API.md)
 
 # Running the project
 
+Note: Use node `v18.15.0`
```
Comment (prichodko) [inline README.md:11]: If this is a requirement, it should be set as `engines` in root `package.json`.

**Review by prichodko — APPROVED** [review]
👍

**Pull request merged**